### PR TITLE
partially implemented the functionality of displaying data from the db

### PR DESCRIPTION
**Pull request by DianaReviako**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-Django==4.0.1
\ No newline at end of file
+Django==4.0.1
+Pillow
```
Comment (Devdb) [inline requirements.txt:2]: why we are need Pillow ?

**Review by Devdb — COMMENTED** [review]
we do no need data in repo. 
Also we do not need *pyc files

File: requirements.txt (hunk 1 above)
Comment (DianaReviako) [reply requirements.txt:2]: надо

Comment (Devdb) [reply requirements.txt:2]: we need to rework this

**Review by Devdb — APPROVED** [review]

**Pull request merged**